src/skmatter/clustering/_quick_shift.py

```diff
@@ -182,13 +195,13 @@ def _qs_next(
         return next_idx
 
 
-def _get_gabriel_graph(dist_matrix2: np.ndarray):
+def _get_gabriel_graph(dist_matrix_sq: np.ndarray):
```

Comment:
Usually private functions don't need a docstring because they will not rendered in anyway. But since you have them now we can keep them.